packages/types/src/Metadata/v3/Storage.ts

```diff
@@ -25,39 +27,39 @@ export {
 export class DoubleMapType extends Struct {
   constructor (value?: any) {
     super({
-      key1: Text,
-      key2: Text,
-      value: Text,
-      key2Hasher: Text
+      key1: Type,
+      key2: Type,
+      value: Type,
+      key2Hasher: StorageHasher
```

Comment:
Unfortunately key2Hasher is still a Text, revert. Please see https://github.com/paritytech/substrate/blob/master/srml/metadata/src/lib.rs

Reply:
Thank you for the review. When using `Text`, if the type of the key has trait prefix like ’T::AssetId’, I got an error ‘Unknown types found, no types for T::AccountId’. Using `Type` fix the issue. I guess it’s because Type does some format optimise when decoding (such as removing all the trait prefixes `Type._removeTraits()`).

Reply:
Changed type of key2Hasher to Text. Thanks for the suggestion.